src/svcutilcore/tests/SvcutilTests.cs

```diff
@@ -20,10 +20,8 @@ public static void SvcutilTest1()
         {
             string outputFile = Path.GetTempFileName() + ".cs";
             Assert.False(File.Exists(outputFile));
-
-            Tool.Main(new string[] { "/t:xmlserializer", Assembly.GetExecutingAssembly().Location, $"/out:{outputFile}" });
+            Tool.Main(new string[] { Assembly.GetExecutingAssembly().Location, $"/out:{outputFile}" });
             Assert.True(File.Exists(outputFile));
-
             File.Delete(outputFile);
```

Comment:
We have code coverage tooling which generates a report showing you which methods get called. It would be good to compare that report before and after this change to make sure there isn't any code paths which are now getting skipped which we don't want to skip.

Reply:
The code coverage report for svcutil.xmlserializer is empty. What shall we do to enable it?

Reply:
We need to do this as a follow up task as the code coverage run isn't configured to record coverage for svcutil. Can you create an issue for this.

Reply:
Sure.